backend/clickhouse/logs_test.go

```diff
@@ -656,14 +659,16 @@ func TestReadLogsWithKeyFilter(t *testing.T) {
 
 	payload, err = client.ReadLogs(ctx, 1, modelInputs.QueryInput{
 		DateRange: makeDateWithinRange(now),
-		Query:     `service:"image processor" service:"different processor"`,
+		Query:     `service:("image processor" OR "different processor")`,
 	}, Pagination{})
 	assert.NoError(t, err)
 	assert.Len(t, payload.Edges, 2)
 
+	// TODO: Do we want to support this syntax still, or should we force people to
+	// wrap this in a string? `service:"foo:bar"`` if contains a bin_op separator?
```

Comment:
@Vadman97 what do you think about this change? I can probably do something with an error message saying that comparison operators need to be wrapped in quotes or something. Or maybe @mayberryzane has an idea on how to evolve the search grammar to preserve the existing behavior 🤔

Reply:
should be fine as long as the input `colon_delimited:foo:bar` shows an error to the user. perhaps the error / ui could make it clear that wrapping the value in `"` would escape any special characters?